rules/windows/initial_access_first_time_seen_usb_name.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2023/03/16"
+integration = ["endpoint", "windows"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup, New Term"
+min_stack_version = "8.6.0"
```

Comment:
Maybe we can use the same as https://github.com/elastic/detection-rules/pull/2433/files?
```
min_stack_comments = "The New Term rule type used in this rule was added in Elastic 8.4"
min_stack_version = "8.4.0"
```